### ISSUE TITLE
Agentd Error on PhotonOS 1.0

### ISSUE THREAD

**Issue by sbl0101** [monitoringartist/zabbix-docker-monitoring#78]
Im getting this error and the zabbix docker isn't working. 

```
# systemctl status docker-dockbix-agent-xxl.service
● docker-dockbix-agent-xxl.service - Dockbix Agent XXL www.monitoringartist.com
   Loaded: loaded (/usr/lib/systemd/system/docker-dockbix-agent-xxl.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2017-07-24 09:14:17 CEST; 31min ago
 Main PID: 23690 (docker)
   CGroup: /system.slice/docker-dockbix-agent-xxl.service
           └─23690 /usr/bin/docker run --rm -e ZA_Server=Zabbix-IP --log-opt max-size=100m --log-opt max-file=2 --name docker-dockbix-agent-xxl.service -h machinedocker--net=host --privileged -v /:/roo...

Jul 24 09:14:18 [23690]: Freeware licence - Usage of this binary is restricted to official monitoringartist Docker images only.
Jul 24 09:14:18 [23690]: Securing container
Jul 24 09:14:18 [23690]: proot error: execve("/usr/bin/zabbix_agentd"): No such file or directory
Jul 24 09:14:18 [23690]: proot info: possible causes:
Jul 24 09:14:18 [23690]: * the program is a script but its interpreter (eg. /bin/sh) was not found;
Jul 24 09:14:18 [23690]: * the program is an ELF but its interpreter (eg. ld-linux.so) was not found;
Jul 24 09:14:18 [23690]: * the program is a foreign binary but qemu was not specified;
Jul 24 09:14:18 [23690]: * qemu does not work correctly (if specified);
Jul 24 09:14:18 [23690]: * the loader was not found or doesn't work.
Jul 24 09:14:18 [23690]: fatal error: see `proot --help`.
```

I've got this on the systemd fic:

```
[Unit]
Description=Dockbix Agent XXL www.monitoringartist.com
After=docker.service
Requires=docker.service

[Service]
TimeoutStartSec=0
Restart=always
ExecStartPre=/usr/bin/docker pull monitoringartist/dockbix-agent-xxl-limited:latest
ExecStart=/usr/bin/docker run --rm -e "ZA_Server=Zabbix-IP" --log-opt max-size=100m --log-opt max-file=2 --name %n -h %H --net=host --privileged -v /:/rootfs -v /var/run:/var/run monitoringartist/dockbix-agent-xxl-limited:latest
ExecStop=/usr/bin/docker rm -f %n

[Install]
WantedBy=multi-user.target
```

Theres any solution for this problem? Thank you :)

**Comment by jangaraj** [comment]
I know about this problem and I'm solving it. Quick workaround - use previous version `monitoringartist/dockbix-agent-xxl-limited:old` - Bear in mind that `old` is only temporary tag and it may be deleted in the future.

**Comment by sbl0101** [comment]
Now its working with the old image, i'll wait for the solution for the latest one. Thank you :)

**Comment by jangaraj** [comment]
Fixed 3.4 image `monitoringartist/dockbix-agent-xxl-limited:3.4-3beta`. Feel free to test it. Unfortunately, processes metrics (`proc.*`) are not available due to cgroup limitations in this v3 version = limited support for host metrics. I'm waiting for Zabbix 3.4.2 before new Dockbix agent release.
Thank you for your patience.
